packages/@aws-cdk/aws-codecommit/README.md

```diff
@@ -42,17 +47,17 @@ You can also add the Repository to the Pipeline directly:
 const sourceAction = repo.addToPipeline(sourceStage, 'CodeCommit');
 ```
 
-### Events
+## Events
 
-CodeCommit repositories emit CloudWatch events for certain activity.
+AWS CodeCommit repositories emit Amazon CloudWatch events for certain activity.
```

Comment:
`certain activities` perhaps?

Reply:
Good catch. I'll update the docs. BTW, I plan to ask my editor to look at the docs next week. She would have most likely caught this.